recipes/fastlinkingnumbers/build.sh

```diff
@@ -0,0 +1,14 @@
+
+mkdir build
+cd build
+cmake ..
+make
```

Comment:
Have you checked whether the code actually gets build with `-march=native`? (It seems to me from a look at the CMakeLists.txt but I am no cmake expert.) If so, the resulting binary won't work everywhere.

Reply:
(At least that would happen for a Release build.)